framework/src/media/voice/HardwareEndPointDetector.h

```diff
@@ -28,16 +28,18 @@ namespace voice {
 class HardwareEndPointDetector : public EndPointDetector
 {
 public:
-	HardwareEndPointDetector(int card, int device);
+	HardwareEndPointDetector(int normal_card, int normal_device, int sd_card, int sd_device);
 
 public:
 	bool init(uint32_t samprate, uint8_t channels) override;
 	void deinit() override;
 	bool startEndPointDetect(int timeout) override;
 
 private:
-	int mCard;
-	int mDevice;
+	int mNormalCard;
```

Comment:
Could you please add description regarding these variables??

Reply:
I added comments